### ISSUE TITLE
Multi Level XP does not work with Run Effective Energy

### ISSUE THREAD

**Issue by Arktiks** [FarbstoffRSL/RSL-Helper#119]
**Bug describtion / Fehlerbeschreibung**
When you attempt to run _Multi Level XP_ with _Run Effective Energy_ enabled the leveling process finishes when any champion hits max level. This occurs once the autoclicker has started running on effective energy. It seems like the autoclicker swaps to _First Max (except Lead)_ logic after it runs out of energy.

![RunEffectiveEnergyBug](https://user-images.githubusercontent.com/12584593/130451573-a9450184-d536-4d4c-a238-db316f81b852.png)

**HowTo Reproduce / Wie man es reproduziert**
1. Beat the stage you want to farm once normally (e.g. Brutal 12-3)
2. Enable _Multi Level XP_ and set high number of champions to level
3. Enable _Run Effective Energy_
4. Start autoclicker
5. Autoclicker will level champions normally until it runs out of energy
6. After energy runs out autoclicker will start the loop of waiting for energy and replaying level
7. Once any champion hits max level after X loops the status will change to "All champs maxed - done"

**Expected behavior / Erwartetes verhalten**
Autoclicker should continue the _Multi Level XP_ process instead of stopping. 

**System(please complete the following information):**
English
Windows 10 Home 64-bit (10.0, Build 19043)

**Game Info(please complete the following information):**
Shadow Legends 4.50.1
RSL Helper V2.6 - Release 8 - Build: 21

**Additional context / Weiterer Infos**
_Auto Sell Artifacts_ was enabled when testing.
RSL helper does not have admin rights.
Shadow Legends window dimensions: 1400x920

**Comment by BlackHawk3000** [comment]
How long did you wait`?
There may be a delay before the power is updated. 
In addition you write yourself "RSL helper does not have admin rights." the helper NEEDS admin rights to work completely.

And it stops because two of your three champions are already max. so no energy is wasted the helper stops. 

it works as it should

**Comment by Arktiks** [comment]
I waited until it had 10 energy, enough to run the level again. Even while I was writing this ticket the RSL Helper did not change status and replace the max level characters to continue the Multi Level XP feature.

The Multi Level XP feature has worked fine without admin rights so far. I can try enabling the admin rights and see if that fixes the issue.

To further illustrate the Expected Behavior should be that the RSL Helper waits that it has enough energy to run the level. It then replaces the max level champions with fresh champions and replays the level. At least that's how the _Multi Level XP_ feature has worked for me so far. I still had more champions selected in the _Set Food_ dialog.

**Comment by BlackHawk3000** [comment]
Maybe you didn't choose enough heroes.

**Comment by Arktiks** [comment]
I tried to replicate the issue and record it. I managed to record _an error_, but it's a bit different from the How to Reproduce steps. However, it's probably related to the error.

**[RSL-Helper Bug #1](https://www.youtube.com/watch?v=rxv1uj4NwwA)**

Hindsight I should have increased the video playback speed. Notable video timestamps are in the video description. My recording software (Open Broadcaster Software) did not record the Set Food dialog, but I set the Multi Level XP feature to level 20x 1* champions.
